src/extensions/default/bramble/lib/ConsoleManagerRemote.js

```diff
@@ -0,0 +1,29 @@
+(function(transport, global, console) {
+    "use strict";
+
+    function _log(s){
```

Comment:
We need to deal with the fact that the console can take any number of args, and any type of arg--simply saying `s` here is not going to work.  Probably the best thing you can do is to send along an array of all arguments:

```js
function _log() {
  transport.send("bramble-console", Array.from(arguments).slice());
}
```

Now on the receiving side, when you `handleConsoleRequest(msgObj.data)` you'll need to deal with the fact that you have an array of things to pass to the console log:

```js
function handleConsoleRequest(args) {
    // Add an identifier to the front of the args list, so we know this is from us in console
    args = args.unshift("[Bramble Console]");
    // Call console.log, passing our long list of arguments
    console.log.apply(console, args);
}
```